std-bits/tableau/src/main/java/org/enso/tableau/HyperTableColumn.java

```diff
@@ -0,0 +1,53 @@
+package org.enso.tableau;
+
+import com.tableau.hyperapi.Nullability;
+import com.tableau.hyperapi.TableDefinition;
+import com.tableau.hyperapi.TypeTag;
+import java.sql.Types;
+import java.util.OptionalInt;
+
+public record HyperTableColumn(
+    int index,
+    String name,
+    int typeID,
+    boolean nullable,
+    OptionalInt length,
+    OptionalInt precision,
+    OptionalInt scale) {
+  // ** Type ID for JSON data. */
+  public static final int JSON = 10001;
+  // ** Type ID for INTERVAL data. */
+  public static final int INTERVAL = 10002;
+
+  static HyperTableColumn FromHyperColumn(int index, TableDefinition.Column hyperColumn) {
```

Comment:
```suggestion
  static HyperTableColumn fromHyperColumn(int index, TableDefinition.Column hyperColumn) {
```
I think usually our Java naming scheme was that methods start with lowercase, and only types start with uppercase.